lte/gateway/python/integ_tests/s1aptests/util/traffic_util.py

```diff
@@ -112,55 +112,65 @@ def __init__(self):
         self.ue_ipv6_block = "fdee:0005:006c::/64"
         self.agw_ipv6 = "3001::10"
 
-    def exec_command(self, command):
+    def exec_command(self, command, capture_output=False):
         """
         Run a command remotely on magma_trfserver VM.
 
         Args:
             command: command (str) to be executed on remote host
                 e.g. 'sed -i \'s/str1/str2/g\' /usr/local/bin/traffic_server.py'
+            capture_output: Sets the `capture_output` flag in `subprocess.run`
 
         Returns:
-            Shell command execution output
+            Shell command return code and stdout
         """
-        data = self._cmd_data
-        data["command"] = '"' + command + '"'
-        param_list = shlex.split(self._command.format(**data))
-        return subprocess.call(
+        self._cmd_data["command"] = f'"{command}"'
+        param_list = shlex.split(self._command.format(**self._cmd_data))
+        return subprocess.run(
             param_list,
             shell=False,
-            stdout=subprocess.DEVNULL,
-            stderr=subprocess.DEVNULL,
+            capture_output=capture_output,
         )
 
+    def dump_leases(self):
+        """Dump DHCP leases in TRF server VM"""
+        return self.exec_command("dumpleases").stdout
```

Comment:
Why is `capture_output=True` not needed here?

Reply:
Good point, I believe I missed running make again before re-running the tests.